code/modules/surgery/organic_steps.dm

```diff
@@ -2,7 +2,11 @@
 //make incision
```

Comment:
Why would they split generic into two separate .dm's?! That's really annoying to path!